rotkehlchen/globaldb/handler.py

```diff
@@ -1292,18 +1292,20 @@ def _edit_token(
         return entry.identifier
 
     @staticmethod
-    def set_token_protocol_if_missing(token: EvmToken, new_protocol: str) -> None:
-        """Update the protocol of the evm token and clean the resolver cache"""
-        if token.protocol == new_protocol:
+    def set_token_protocol_if_missing(tokens: list[EvmToken], new_protocol: str) -> None:
```

Comment:
```suggestion
    def set_tokens_protocol_if_missing(tokens: list[EvmToken], new_protocol: str) -> None:
```
I think it would be better to have tokens be plural in the function name as well - otherwise it sounds like it should only take a single token instead of a list.